xdggs/accessor.py

```diff
@@ -115,3 +116,40 @@ def cell_boundaries(self):
         return xr.DataArray(
             boundaries, coords={self._name: self.cell_ids}, dims=self.cell_ids.dims
         )
+
+    def explore(
+        self, *, cell_boundaries="cell_boundaries", cmap="viridis", center=None
+    ):
+        """interactively explore the data using `lonboard`
+
+        Requires `geopandas`, `matplotlib`, and `lonboard` to be installed.
+
+        Parameters
+        ----------
+        cell_boundaries : str
+            The name of the coordinate containing the pre-computed cell boundary polygons.
+        cmap : str
+            The name of the color map to use
+        center : int or float, optional
+            If set, will choose this as the center point for a diverging color map.
+
+        Returns
+        -------
+        map : lonboard.Map
```

Comment:
It doesn't look like you have docs yet, but if you use sphinx or mkdocs (with mkdocstrings), lonboard implements [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html), so you should be able to set it up so the return value is a hyperlink to the lonboard docs.

Reply:
thanks for the hint. I'll probably use `sphinx` to set up the docs, so I'll definitely make use of that!